spki/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "spki"
-version = "0.7.3"
+version = "0.8.0-pre"
```

Comment:
If you don't mind I'd love to include https://github.com/RustCrypto/formats/pull/1290 in there as well.

Reply:
You can just rebase/merge that after I land this, which is only bumping versions.

Reply:
Not yet (though I guess I usually note there's not a release associated with these PRs and forgot in this case).

This is just bumping versions to signal that breaking changes are coming.